lib/src/utils/alignments.dart

```diff
@@ -14,3 +14,13 @@ enum CrossGroupAlignment {
   center,
   end,
 }
+
+/// How the button runs themselves should be placed the cross axis in a layout
+enum CrossGroupRunAlignment {
```

Comment:
Please refactor this name **CrossGroupRunAlignment** -> **GroupRunAlignment**

Reply:
Done with the refactoring. Thank you, Waiting for the version release! 🙌